backend/db/migrate/20250816111712_remove_json_data_from_companies.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveJsonDataFromCompanies < ActiveRecord::Migration[8.0]
+  def change
+    remove_column :companies, :json_data, :jsonb
```

Comment:
```suggestion
    remove_column :companies, :json_data, :jsonb, default: {"flags" => []}, null: false
```

To make a rollback add the correct column.